returnn/torch/engine.py

```diff
@@ -416,7 +423,9 @@ def _create_data_loader(self, dataset: Dataset) -> DataLoader2:
         )
 
         wrapped_dataset = returnn_dataset_wrapper.ReturnnDatasetIterDataPipe(dataset, reset_callback=dataset_reset)
-
+        wrapped_dataset = data_pipeline.LenFilterDataPipe(
```

Comment:
For simplicity of the data pipeline, I would only apply this if `_min_seq_len` or `_max_seq_len` is set.